<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformationPane</class>
 <widget class="QWidget" name="InformationPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_FilmScreen">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_FilmScreen">
        <property name="text">
         <string>Affiche du film :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="posterComboBox"/>
      </item>
      <item>
       <widget class="QGraphicsView" name="posterGraphicsView"/>
      </item>
      <item>
       <widget class="QLabel" name="label_FilmScreenDefault">
        <property name="text">
         <string>Capture par defaut :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sampleComboBox"/>
      </item>
      <item>
       <widget class="QGraphicsView" name="sampleGraphicView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Files">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QTreeView" name="treeView_Files"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
